app/helpers/application_helper.rb

```diff
@@ -23,7 +23,12 @@ def avatar(email = nil)
     else
       gravatar_id = Digest::MD5.hexdigest(email.downcase)
       gravatar_size = Rails.application.config.gravatar_size
-      tag :image, src: "http://gravatar.com/avatar/#{gravatar_id}.png?s=#{gravatar_size}&d=#{CGI.escape(root_url + default)}"
+      if Rails.env.development? 
```

Comment:
Is it really necessary to do this check on every request. IMHO, we should keep that decision only in initialization and use the variable directly here.
i.e: make gravatar_default defined in 0_glitterconfig.rb
what do you think @rohitpaulk ?